lib/transpose_cases.rb

```diff
@@ -0,0 +1,45 @@
+class TransposeCase < OpenStruct
+  def test_name
+    "test_#{description.tr(' ', '_')}"
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def workload
+    'Transpose.transpose(input)'
+  end
+
+  def expect
+    [
+      "<<-EXPECTED.gsub(/^ {6}/, '')",
+      indent_lines(expected),
+      indent_line('EXPECTED')
```

Comment:
For this to work I will have to monkey patch the String class with a `left_trim` method in the generated tests?